plugins/inventory/xen_orchestra.py

```diff
@@ -0,0 +1,315 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2021 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    name: community.general.xen_orchestra
```

Comment:
```suggestion
    name: xen_orchestra
```
This one needs to stay short. (It's the only one :) )